queries.sql

```diff
@@ -119,3 +119,62 @@ LIMIT 1;
 
 
 
+-- JOIN Table Queries
+
+SELECT animals.name FROM visits
+JOIN vets ON vet_id = vets.id
+JOIN animals ON animal_id = animals.id
+where vets.name = 'William Tatcher'
+ORDER BY date_of_visit DESC
+LIMIT 1;
+
+SELECT COUNT(date_of_visit) FROM visits
+JOIN vets ON vet_id = vets.id
+WHERE vets.name = 'Stephanie Mendez';
+
+SELECT vets.name, species.name FROM vets
+LEFT JOIN specializations ON vets.id = vet_id
+LEFT JOIN species ON species.id = species_id;
+
+SELECT animals.name FROM visits
+JOIN animals ON animal_id = animals.id
+JOIN vets ON vet_id = vets.id
+WHERE vets.name = 'Stephanie Mendez'
+AND date_of_visit BETWEEN '2020-04-01' AND '2020-08-30';
+
+SELECT animals.name FROM visits
+JOIN animals ON animal_id = animals.id
+GROUP BY animals.name
+ORDER BY COUNT(animals.id) DESC
+LIMIT 1;
+
+SELECT animals.name FROM visits
+JOIN animals ON animal_id = animals.id
+JOIN vets ON vet_id = vets.id
+WHERE vets.name = 'Maisy Smith'
+ORDER BY date_of_visit
+LIMIT 1;
+
+SELECT animals.name, vets.name, date_of_visit FROM visits
+JOIN animals ON animal_id = animals.id
+JOIN vets ON vet_id = vets.id
+ORDER BY date_of_visit DESC
+LIMIT 1;
+
+SELECT COUNT(*) FROM visits
+JOIN animals ON animal_id = animals.id
+JOIN vets ON vet_id = vets.id
+JOIN specializations ON vets.id = specializations.vet_id
+WHERE animals.species_id != specializations.species_id;
+
```

Comment:
- [ ] I have observed in your screenshots that this query is returning 14 records but if we manually count then there are only 12 visits which are done by non-specialized vets in those species 😨 It might be an issue of your inserted data or this query i am highlighting so kindly fix this query as well or fix your inserted data to fix this issue to make your query return the correct data 👍